spyder/widgets/shortcutssummary.py

```diff
@@ -106,6 +107,11 @@ def __init__(self, parent=None):
                 # Widgets
                 label_name = QLabel(name.capitalize().replace('_', ' '))
                 label_name.setFont(font_names)
+
+                if sys.platform.startswith('darwin'):
+                    keystr = keystr.replace('Ctrl',
+                                            'Cmd').replace('Alt', 'Option')
```

Comment:
Does <kbd>Meta</kbd> already get replaced with <kbd>Control</kbd> on macOS? Also, what about <kbd>Backspace</kbd> to <kbd>Delete</kbd>? Presumably, @jnsebgosselin 's suggestion would also resolve this.

Reply:
But presuming it does, you'd want to include examples of each in your test.